src/app/space/create/deployments/apps/deployment-card-container.component.ts

```diff
@@ -14,7 +16,22 @@ import { Environment } from '../models/environment';
 })
 export class DeploymentCardContainerComponent {
   @Input() spaceId: string;
+  @Input() spaceName: string;
   @Input() environments: Observable<string[]>;
-  @Input() application: string;
-  collapsed: boolean = true;
+  @Input() applicationsList: string[];
+  collapsed: {[applicationName: string]: boolean} = {};
```

Comment:
Can a Map be used here rather than an array? Or does that make accessing it from within the template a lot uglier?